content/en/blog/2023/announcing-notation-rc5.md

```diff
@@ -0,0 +1,60 @@
+* Use SHA256 instead of insecure SHA1 for certificate thumbprint in `notation inspect`
+* Bump github.com/opencontainers/image-spec from 1.1.0-rc2 to 1.1.0-rc.3
+
+Notation-go is a collection of Golang libraries to support signing and verifying OCI artifacts. It is based on Notary standard. Notation-go v1.0.0-RC.5 removes OCI artifact manifest type from [Sign](https://pkg.go.dev/github.com/notaryproject/notation-go@main#Sign) function.
+
+### Deprecate flag `--signature-manifest` in notation sign
+
+Since v1.0.0-RC.5, Notation no longer supports create signatures with OCI artifact manifest type in the signing process and deprecated the flag `--signature-manifest` from `notation sign`. Please note this is a breaking change in Notation v1.0.0-RC.5. We chose to make this deprecation before the stable v1 release due to the upstream [OCI image-spec v1.1.0-rc.3](https://github.com/opencontainers/image-spec/releases/tag/v1.1.0-rc.3) removed OCI artifact manifest.
+
+To mitigate the impact to Notation users, Notation v1.0.0-RC.5 is still able to read and verify the existing signatures stored with the OCI artifact manifest type in OCI v1.1.0-RC.1 compliant registries. If you upgrade Notation to v1.0.0-RC.5 from a previous RC release, you don't need to concern too much about the OCI artifacts signed by Notation as they are still verifiable.
+
+### Change default behavior and introduce a new flag `--allow-referrers-api`
+
+As Notation is close to the first stable v1 release, we are trying to make Notation as stable as possible and avoid breaking changes going forward. Considering the OCI Image Spec and Distribution Spec are still in RC status, the [Referrers Tag Schema](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#referrers-tag-schema) is relatively stable and potable so it has been used as the default behavior since Notation v1.0.0-RC.5. Most of the registries are compatible with the tag described by the Referrers Tag Schema. However, the [Referrers API](https://github.com/opencontainers/distribution-spec/blob/main/spec.md#enabling-the-referrers-api) and its fallback strategy is relatively instable according to recent OCI Distribution Spec changes, so it is marked as experimental in Notation v1.0.0-RC.5.
+
+In this context, Notation v1.0.0-RC.5 introduces a new experimental flag `--allow-referrers-api` to the following commands:
+
+- notation sign
+- notation verify
+- notation inspect
+- notation list
+
+Specifically, users need to set the environment variable `NOTATION_EXPERIMENTAL=1` to enable this new flag, then Notation attempts the Referrers API and fallback to Referrers Tag Schema on failure when using with OCI registry. Otherwise, Notation has deterministic behavior and uses Referrers Tag Schema by default.
```

Comment:
I am not sure every reader will know what _"Referrers Tag Schema"_ means and what the impact to them is. Can we add more clarification to that.

Reply:
I added a note to explain the concept of the Referrers Tag Schema. See [preview](https://deploy-preview-218--notarydev.netlify.app/blog/2023/announcing-notation-rc5/#change-default-behavior-and-introduce-a-new-flag---allow-referrers-api)

![image](https://github.com/notaryproject/notaryproject.dev/assets/40452856/58665118-ebaa-4be9-b2c5-e49128b0a12d)